packages/default/scss/scrollview/_layout.scss

```diff
@@ -1,154 +1,175 @@
+            bottom: $scrollview-pager-offset;
+            pointer-events: none;
         }
-    }
 
-    .k-pages,
-    .k-scrollview-pageable {
-        margin: 0 auto;
-        padding: 0;
-        width: 100%;
-        display: flex;
-        justify-content: center;
-        text-align: center;
-        list-style: none;
-        position: absolute;
-        left: 0;
-        bottom: $scrollview-pager-offset;
-        pointer-events: none;
-    }
-    .k-pages > li,
-    .k-scrollview-pageable > li.k-button {
-        margin: 0 $scrollview-pager-item-spacing;
-        padding: 0;
-        width: $scrollview-page-button-size;
-        height: $scrollview-page-button-size;
-        box-sizing: content-box;
-        vertical-align: middle;
-        display: inline-block;
-        flex: 0 0 $scrollview-page-button-size;
-        border-radius: 50%;
-        cursor: pointer;
-        pointer-events: all;
-    }
+        .k-scrollview-pageable > li.k-button,
+        .k-scrollview-nav > li.k-link {
+            margin: 0 $scrollview-pager-item-spacing;
+            padding: 0;
+            width: $scrollview-pagebutton-size;
+            height: $scrollview-pagebutton-size;
+            box-sizing: content-box;
+            vertical-align: middle;
+            display: inline-block;
+            position: relative;
+            flex: 0 0 $scrollview-pagebutton-size;
+            border-width: $scrollview-pager-item-border-width;
+            border-style: solid;
+            border-radius: 50%;
+            cursor: pointer;
+            pointer-events: all;
+
+            &::before {
+                content: "";
+                width: $scrollview-pagebutton-size + $scrollview-pager-item-spacing;
+                height: $scrollview-pager-height;
+                display: block;
+                position: absolute;
+                top: 50%;
+                left: 50%;
+                transform: translate(-50%, -50%);
+            }
+        }
 
-    .k-scrollview-next,
-    .k-scrollview-prev {
-        display: table;
-        position: absolute;
-        padding: 0;
-        height: 60%;
-        top: 20%;
-        text-decoration: none;
-        user-select: none;
-        cursor: pointer;
-        overflow: hidden;
-        -webkit-tap-highlight-color: $rgba-transparent;
-    }
+        //remove when https://github.com/telerik/kendo-angular-scrollview/issues/130 is done
+        .k-scrollview-pageable > li.k-button::before {
+            pointer-events: initial;
+            border-radius: 0;
+        }
 
-    .k-scrollview-prev {
-        left: 0;
-    }
+        .k-scrollview-next,
+        .k-scrollview-prev {
+            display: table;
+            position: absolute;
+            padding: 0;
+            height: 60%;
+            top: 20%;
+            text-decoration: none;
+            user-select: none;
+            cursor: pointer;
+            overflow: hidden;
+            -webkit-tap-highlight-color: $rgba-transparent;
```

Comment:
[suggestion] consider exposing a variable for customization of this color, if not needed, ignore this comment